src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -537,9 +513,6 @@ private StaticSelectElement getLicenseTypeElement(UserDTO userDTO) {
         if (withDuplicateUserClarificationNote(userDTO, actionId)) {
             layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking why they registered multiple accounts.", DUPLICATE_USER_CLARIFICATION_NOTE));
         }
-        if (isMSKUser(userDTO)) {
```

Comment:
I think we still need this, but rather to apply to all regular commercial companies.